2-mandatory/1-create-functions.js

```diff
@@ -44,7 +46,13 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  const newArr = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > 100) arr[i] = 100;
+    newArr.push(Math.round(arr[i] * 100) / 100 + "%");
```

Comment:
As you are returning a new array, I dont think you need to `arr[i] = 100` which modifies the original array.